wowup-electron/src/app/components/wow-client-options/wow-client-options.component.html

```diff
@@ -50,6 +50,15 @@
         (change)="onDefaultAutoUpdateChange($event)">
       </mat-slide-toggle>
     </div>
+    <div class="row">
```

Comment:
Lets maybe move this up to be the first option under the file path just so it seems important